includes/legacy/class-wcs-product-legacy.php

```diff
@@ -30,11 +30,6 @@ public static function init() {
 	 * @since 1.0.0 - Migrated from WooCommerce Subscriptions v2.2.0
 	 */
 	public static function set_product_class( $classname, $product_type, $post_type, $product_id ) {
-
-		if ( wcs_is_woocommerce_pre( '3.0' ) && in_array( $classname, array( 'WC_Product_Subscription', 'WC_Product_Variable_Subscription', 'WC_Product_Subscription_Variation' ) ) ) {
-			$classname .= '_Legacy';
-		}
-
 		return $classname;
 	}
```

Comment:
This `set_product_class()` function is not required now because it's a filter hooked onto `woocommerce_product_class` and it now just straight up returns the classname input. 

That then brings into question the need for this class at all as this was the only function in this class. I'll file a separate issue to follow up on this though because that's a larger set of changes than this PR should be responsible for.